stable/jenkins/README.md

```diff
@@ -236,6 +236,31 @@ If running upon a cluster with RBAC enabled you will need to do the following:
 * Create a Jenkins credential of type Kubernetes service account with service account name provided in the `helm status` output.
 * Under configure Jenkins -- Update the credentials config in the cloud section to use the service account credential you created in the step above.
 
+## Backup
+
+Adds a backup CronJob for jenkins, along with required RBAC resources.
+
+### Backup Values
+
+| Parameter                   | Description                                | Default                           |
+| --------------------------- | ------------------------------------------ | --------------------------------- |
+| `backup.enabled`            | Enable the use of a backup CronJob         | `false`                           |
+| `backup.schedule`           | Schedule to run jobs                       | `0 2 * * *`                       |
+| `backup.annotations`        | Backup pod annotations                     | iam.amazonaws.com/role: `jenkins` |
+| `backup.image.repo`         | Backup image repository                    | `nuvo/kube-tasks`                 |
```

Comment:
This image needs to be added to sources.

Reply:
done